pkg/utils/ocmWrapper.go

```diff
@@ -222,6 +224,55 @@ func (*DefaultOCMInterfaceImpl) GetBackplaneURL() (string, error) {
 	return "", nil
 }
 
+// Set Backplane Proxy URL from local settings
+func (*DefaultOCMInterfaceImpl) SetBackplaneProxyURL(proxyUrl string) error {
```

Comment:
We can remove this

Reply:
Ack .. Shall I move this proxy URL deriving logic to the utility function then?

Reply:
done